biogrid/libs/biogrid-simulator/src/lib/biobattery/biobattery-class.ts

```diff
@@ -0,0 +1,62 @@
+import { Battery, Energy, Validatable, validate } from "@biogrid/grid-simulator";
+
+export class BioBattery implements Battery {
+  private currentBatteryPower: Energy;
+  private readonly maxCapacity: Energy;
+
+  constructor(currentBatteryPower: Energy = 0,maxCapacity: Energy = 10) {
+    
+    if(!this.validateInputs(currentBatteryPower, maxCapacity)) {
+      throw new Error('Cannot create a battery with these values');
+    }
+    this.currentBatteryPower = currentBatteryPower;
+    this.maxCapacity = maxCapacity;
+  }
+
+  startCharging(inputPower: Energy): void {
+    if (this.currentBatteryPower + inputPower > this.maxCapacity) {
+
```

Comment:
what's supposed to happen in this if statement?

Reply:
> what's supposed to happen in this if statement?
Forgot to write the condition, I updated it